templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -234,7 +234,11 @@ export class {{ service.name }}Client {
         '{{ field }}'
 {%- endfor -%}
         ],
+{%- if method.bundleConfig.batchDescriptor.subresponse_field === 'null' %}
         {{- method.bundleConfig.batchDescriptor.subresponse_field }},
```

Comment:
OK so here we know it's `null`, so maybe just say `null`? :)

```
{%- if method.bundleConfig.batchDescriptor.subresponse_field === 'null' %}
        null,
```

Reply:
makes sense!